gradle/dependencies.gradle

```diff
@@ -77,7 +77,7 @@ versions += [
   jersey: "2.28",
   jmh: "1.21",
   hamcrest: "2.1",
-  log4j: "1.2.17",
+  log4j: "2.12.1",
```

Comment:
Based on the comments, how about `2.13.x`?

Reply:
@cricket007 Yes, I am now working with `2.13.1` and it seems like good.